packages/sitecore-jss-proxy/src/index.ts

```diff
@@ -384,9 +384,10 @@ export function rewriteRequestPath(
   let finalReqPath = decodedReqPath;
```

Comment:
My previous comment was about line 373 here, where `isUrlIgnored(reqPath, config)` is called.